unity-client/Assets/Scripts/MainScripts/DCL/Helpers/UIHelpers/RawImageFillParent.cs

```diff
@@ -0,0 +1,53 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class RawImageFillParent : RawImage
```

Comment:
Can we add a bit of documentation about what this helper class is supposed to do?

Reply:
added summary comment